packages/catalog-model/examples/acme/org.yaml

```diff
@@ -3,6 +3,11 @@ kind: Group
 metadata:
   name: acme-corp
   description: The acme-corp organization
+  links:
+    - url: http://www.acme.com/
+      name: Website
```

Comment:
Title, not name, ya?

Reply:
😢 Where is my head today ...

Reply:
Pixies! https://www.youtube.com/watch?v=49FB9hhoO6c